kore/kore.cabal

```
@@ -157,7 +157,8 @@ common library
   build-depends: ghc-trace-events >=0.1
   build-depends: gitrev >=1.3
   build-depends: graphviz >=2999.20
-  build-depends: hashable >=1.2
+  -- Pin hashable to make sure cabal build uses the one Nix & Stack builds do
+  build-depends: hashable >=1.3.0.0
```

Not sure this is sufficient. The [`cabal` build in CI is using `1.3.5.0`](https://github.com/runtimeverification/haskell-backend/runs/7285590597?check_suite_focus=true) while the [`stack` build is using `1.3.0.0`](https://github.com/runtimeverification/haskell-backend/runs/7285591760?check_suite_focus=true) (as per `lts-18.18` resolver). This should probably be pinned to _exactly_ `1.3.0.0` ?
```suggestion
  build-depends: hashable ==1.3.0.0
```

---

Thank you for the heads up, this was exactly my intent, but somehow the `>=` snuck in